src/extension/wps/wps-core/src/main/java/org/geoserver/wps/gs/ImportProcess.java

```diff
@@ -280,8 +282,8 @@ else if (coverage != null) {
         else if (coverage != null)
         {
         	try {
-        		final File directory = catalog.getResourceLoader().findOrCreateDirectory("data", workspace, store);
-        		final File file = File.createTempFile(store, ".tif", directory);
+        		final Resource directory = catalog.getResourceLoader().get(Paths.path("data", workspace, store));
+        		final File file = File.createTempFile(store, ".tif", directory.dir());
```

Comment:
@bencaradocdavies No problem, but mind that this method is huge and it would make it impossible to see what I actually changed.

In my early days of geotools/geoserver I was once told off with one of my first patches that I had been fixing wonky formatting and that obscured the real changes I did, and that formatting fixes should be in separate patches. Since then I avoid doing that.

As a side note, perhaps we should just fix all of these formatting errors in an automised manner across the whole codebase? That would be more time efficient for both programmer and reviewer.